feat: add the money talk notes (#40)
cybermagnolia/cybermagnolia.com: content/blog/the-money-talk-meetup.md

### Appreciate The Role Of Money

Treating salary like pocket money that we receive at our employer's whim
happens to more women than we'd like to admit. Very few of us think of
ourselves as the providers. This doesn't seem to be rooted in reality
though---casual chats quickly reveal how many women have at some point
used their money to support an unemployed spouse, their parents or even
elderly grandparents, not to mention their children.

Owning the role of the provider and a financially independent person
helps change perspective. Making your negotiation process about your
ability to support your loved ones is a powerful way to strengthen your
own resolve when it gets stressful.

### Focus On Data

A good way to help detach from the emotions and focus on facts in any
finance-related discussion is using data. Understand what are the
current rates for your type of role in the market you work in. Do solid
research (we'll talk about it in the next step) and treat this
discussion as any other merit-based professional conversation that you
prepare for.

### Accept The Process

There's an important point you need to acknowledge---negotiating salary

> is part of the process of finding a new job or raising to the next level

Reviewer: ```suggestion
is a part of the process of finding a new job or raising to the next level
```

in your current organization. There is an asymmetry of information in
the professional context because your employer knows its own budget and
salaries of everyone employed at the company, but you usually don't. In

> that situation the company is benefiting from the lack of transparency

Reviewer: ```suggestion
that situation, the company is benefiting from the lack of transparency
```

and if they wouldn't want salary negotiations to be a part of the
process, they'd introduce some level to transparency (e.g. by providing
official salary ranges.)

## Do The Research

Figuring out the value of your work compared to the current market
situation is a necessary starting point. Here's how to make sure you get

> most correct data:

Reviewer: ```suggestion
the most correct data:
```

interviewing for, a more senior person in the same industry and job role
as you, or a colleague from your own company if you're looking to
negotiate a raise. Ask a couple of different people to get a better
overview of the estimates.

## Talk The Numbers

There are certain points at which you can expect the salary question to
pop up. In the Czech market it is commonplace for recruiters to ask for
the salary range already in the screening conversation, before you even
talk to the team. Be prepared for this moment.

### Impact Of Negotiation On The Work Relationships

Bear in mind that the process of negotiation may affect the quality of
the relationship with your future managers. Literature describes a
phenomenon known as ["the social cost of
negotiating"](https://women.govt.nz/nacew/information/develop-your-workplace-negotiation-skills/why-negotiation-issue-women/social-costs)---it's
important to know how to phrase the discussion to appear [less
combative](https://hbr.org/2014/06/why-women-dont-negotiate-their-job-offers).
Make sure to watch [Margaret Neale's (Stanford Graduate School of
Business)
video](https://www.youtube.com/watch?v=MXFpOWDAhvM) about

> framing negotiation as a problem solving exercise with the focus on

Reviewer: ```suggestion
framing negotiation as a problem-solving exercise with the focus on
```

benefits to your counterparts, team, or organization.

One thing to keep in mind---don't let this issue get in the way of
negotiating your salary. US research shows that giving up on negotiation
can cost Americans [1 million dollars over the span of their
career](https://www.businessinsider.com/not-negotiating-costs-workers-1-million-2013-9).
Now, a million dollars is a good motivation to overcome the negotiation
stress, isn't it?

### When Does It Make Sense To Talk About Numbers

> The answer to that question is unfortunately "It depends." While a bulk

Reviewer: ```suggestion
The answer to that question is, unfortunately, "It depends." While a bulk
```

of negotiation advice states it's a mistake to reveal your numbers
first, there's some merit to using the ["anchoring"
technique](https://hbswk.hbs.edu/archive/when-to-make-the-first-offer-in-negotiations).
Make sure to filter any advice on this topic through the cultural
lens---US culture tends to be more accepting of aggressive negotiation,
while many European cultures leave little room for it.

If you're a career changer or for any other reason your CV may be less
impressive than your full skillset, then it really makes sense to leave
this conversation for the end. Your strong performance in the interview
will allow you to present yourself as a solid candidate.

It's fair to explain that you'd like to fully understand the range of
responsibilities and the details of the job before estimating your
salary for that particular role. If the person asking for the financial
expectations keeps pushing, you can counter by asking "Are you able to
share the salary ranges that you have for that position with me?" At

> that point most people agree to discuss the numbers after the interview

Reviewer: ```suggestion
that point, most people agree to discuss the numbers after the interview
```

round.

### Ranges Versus Single Data Point

It appears that [providing a range of numbers may be better for the
financial outcome as well as the relationship with the other
party](https://www.alexgozdek.com/is-it-better-to-offer-a-salary-range-or-a-single-number-during-negotiations/).
It gives an impression of more flexibility and goodwill in the process.
Most employers will avoid going below the lowest number (because it's
understood that it's a no-go for you) and try to hit the middle range.

It makes sense to set the lower bound of the range to a number that you
would be happy to accept, and set the upper bound either at 10-20%
higher, or if you're feeling confident in the particular negotiation
even a bit more. It seems there's a lower relationship penalty for
setting a more aspirational upper bound as long as the lower bound is
reasonable.

### Make Sure You Always Have An Alternative

In addition to having solid research to back up your ask, it's necessary
to evaluate your alternative options to establish what's known as
[BATNA](https://www.storyboardthat.com/articles/b/best-alternative-to-negotiated-agreement)---Best

> Alternative To Negotiated Agreement. To put it simply you need to know

Reviewer: ```suggestion
Alternative To Negotiated Agreement. To put it simply, you need to know
```

> what are your options if the negotiation will fail. Can you simply stay

Reviewer: ```suggestion
what are your options if the negotiation fails. Can you simply stay
```

can include learning budgets (to cover courses, conference attendance,
books, etc), extra vacation days, parental benefits, private health
care. If any of those are important to you, it makes sense to discuss
those benefits during the negotiation phase.

If the offer you're reviewing comes from a [publicly
traded](https://en.wikipedia.org/wiki/Public_company)
company or a startup, it makes sense to discuss stock as part of your
compensation package. It's particularly important to understand
[vesting cliffs and
schedules](https://www.investopedia.com/terms/c/cliffvesting.asp).

### Accepting A Counteroffer

It may happen that when you hand in your resignation your current
employer will come back to you with a counteroffer---a promise to raise
your salary to keep you in the team. Make sure to [think it
through](https://www.apollotechnical.com/reasons-to-not-accept-a-counteroffe)
before accepting the counter. While everyone's circumstances differ, the
potential relief caused by the thought that you won't have to go through
the stress of changing workplaces frequently isn't worth it. The
original issues that made you decide to job hunt will not be

> autoresolved, and there may be some negative feelings on both sides.

Reviewer: ```suggestion
auto resolved, and there may be some negative feelings on both sides.
```

### Finding A Job That Values Your Full Skillset

If you receive a job offer with a salary far below what you feel is
fair, and the employer isn't willing to meet your numbers, it's
perfectly fine to walk away from the offer. It may happen that your
skillset is not the right fit for their role, or they are looking for

> just a fraction of what you can bring to the new team. In that case you

Reviewer: ```suggestion
just a fraction of what you can bring to the new team. In that case, you
```
